dashboard/modules/reporter/reporter_head.py

```diff
@@ -77,8 +77,8 @@ async def get_ray_config(self, req) -> aiohttp.web.Response:
                     message="Invalid config, could not load YAML.")
 
             payload = {
-                "min_workers": cfg["min_workers"],
-                "max_workers": cfg["max_workers"]
+                "min_workers": cfg.get("min_workers", "unspecified"),
+                "max_workers": cfg.get("max_workers", "unspecified")
```

Comment:
but we still use the global max_workers though.

Reply:
I agree, but I think that stating 'unspecified' is better than this method erroring!

Reply:
but it should not error though. If it errors then we have a bug. Am I missing something??

Reply:
I guess for `max_workers`, yes, but at the end of the day we are **parsing a user-specified file**. I would always rather err on the side of caution here.